libparsec/crates/types/src/local_manifest.rs

```diff
@@ -278,36 +293,49 @@ impl LocalFileManifest {
     pub fn assert_integrity(&self) {
```

Comment:
Can you create an issue about this not having the right type ?
(this is rather important given it means any bad data simply crashes the application...)

Reply:
https://github.com/Scille/parsec-cloud/issues/6599